.github/workflows/on-release.yaml

```diff
@@ -119,7 +119,7 @@ jobs:
       - container-whalesay
       - container-whalesay2
       - chart-helloworld
-    if: fromJSON(needs.changes.outputs.changes).workflows == 'true'
+    if:  ${{ always() && ((needs.container-whalesay.result == 'skipped' || needs.container-whalesay.result == 'success') && (needs.container-whalesay2.result == 'skipped' || needs.container-whalesay2.result == 'success') && (needs.chart-helloworld.result == 'skipped' || needs.chart-helloworld.result == 'success') && fromJSON(needs.changes.outputs.changes).workflows == 'true') }}
```

Comment:
We can refactor this into a `contains()` expression on `needs.*.result` like we used to do for the slack-update task at the end of the workflows.